diff --git a/finetune_multigpu.py b/finetune_multigpu.py
index b3d514a..0a6443f 100644
--- a/finetune_multigpu.py
+++ b/finetune_multigpu.py
@@ -277,7 +277,6 @@ def finetune(model_path:str, model_size: str, num_fingerprints: int, max_key_len
                             'zero_optimization': {
                                                 'stage': 2, 
                                                     'offload_optimizer': {'device': 'cpu', 'pin_memory': True},
-                                                    'offload_param': {'device': 'cpu', 'pin_memory': True},
 
 
                                                 }
